From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Mon, 15 Apr 2013 09:47:17 +0100
Subject: disable sml-mode-expansions

The sml-mode support was tested with sml-mode 6.3, and does not seem
compatible with the version in Debian, sml-mode 4.1-x.
---
 expand-region.el | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/expand-region.el b/expand-region.el
index a568806..4fc8093 100644
--- a/expand-region.el
+++ b/expand-region.el
@@ -185,7 +185,9 @@ before calling `er/expand-region' for the first time."
 (eval-after-load "cc-mode"      '(require 'cc-mode-expansions))
 (eval-after-load "text-mode"    '(require 'text-mode-expansions))
 (eval-after-load "cperl-mode"    '(require 'cperl-mode-expansions))
-(eval-after-load "sml-mode"     '(require 'sml-mode-expansions))
+;; This was tested w/ sml-mode 6.3.
+;; Does not work with Debian sml-mode 4.1-x.
+;;(eval-after-load "sml-mode"     '(require 'sml-mode-expansions))
 
 (provide 'expand-region)
 
